paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/TransformToBankIcon.kt

```diff
@@ -20,7 +20,7 @@ internal object TransformToBankIcon {
             Regex("PNC\\s?BANK|PNC Bank", RegexOption.IGNORE_CASE) to "pnc",
             Regex("SUNTRUST|SunTrust Bank", RegexOption.IGNORE_CASE) to "suntrust",
             Regex("Silicon Valley Bank", RegexOption.IGNORE_CASE) to "svb",
-            Regex("Stripe|TestInstitution", RegexOption.IGNORE_CASE) to "stripe",
+            Regex("Stripe|TestInstitution|Test Institution", RegexOption.IGNORE_CASE) to "stripe",
```

Comment:
So that we show the Stripe logo instead of the plain bank logo for test accounts.